superset/tags/commands/create.py

```diff
@@ -60,3 +62,45 @@ def validate(self) -> None:
             )
         if exceptions:
             raise TagInvalidError(exceptions=exceptions)
+
+
+class CreateCustomTagWithRelationshipsCommand(CreateMixin, BaseCommand):
+    def __init__(self, data: dict[str, Any]):
+        self._tag = data["name"]
+        self._objects_to_tag = data.get("objects_to_tag")
+        self._description = data.get("description")
+
+    def run(self) -> None:
+        self.validate()
+        try:
+            tag = TagDAO.get_by_name(self._tag.strip(), TagTypes.custom)
+            if self._objects_to_tag:
+                TagDAO.create_tag_relationship(
+                    objects_to_tag=self._objects_to_tag, tag=tag
+                )
+
+            if self._description:
+                tag.description = self._description
+                db.session.commit()
+
+        except DAOCreateFailedError as ex:
+            logger.exception(ex.exception)
```

Comment:
flask will actually raise and log this error from the response. I don't think we need this.